gtpv2/message/update-bearer-res_test.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019-2021 go-gtp authors. All rights reserved.
+// Use of this source code is governed by a MIT-style license that can be
+// found in the LICENSE file.
+
+package message_test
+
+import (
+	"testing"
+
+	"github.com/wmnsk/go-gtp/gtpv2/message"
+	"github.com/wmnsk/go-gtp/gtpv2/testutils"
+)
+
+func TestUpdateBearerResponse(t *testing.T) {
+	cases := []testutils.TestCase{
+		{
+			Description: "Normal",
+			Structured: message.NewUpdateBearerResponse(
+				testutils.TestBearerInfo.TEID, testutils.TestBearerInfo.Seq,
+			),
+			Serialized: []byte{
+				// Header
```

Comment:
Only header? What about other fields?

Reply:
If we look at other's UT, there are some places where the only Header is tested but in messages are a lot of IE's. So as I understand, we can do it later or on PR into the main repo if needed.

Also I need some examples of live traffic with UpdateBearer, but I can't find them.

Reply:
Don't think it is a good move to take the worst case as reference point. 
You could get these bytes on your own by serializing struct, ensure they are fine and cement with UT by extending Serialized here.

Reply:
Done in https://github.com/1NCE-GmbH/go-gtp/pull/9/commits/bbd2d9396dac2c1dd275c9e01b38792a88ca4a22